spec/spec_helper.rb

```diff
@@ -1,5 +1,17 @@
 require 'simplecov'
 require 'simplecov-console'
+require 'pg'
```

Comment:
Gemfiles can be split into environments. Ensure that all test related gems are in test group, e.g. capybara etc.